pkgs/top-level/perl-packages.nix

```diff
@@ -64,7 +64,6 @@ let
       homepage    = http://betterthangrep.com/;
       license     = licenses.artistic2;
       maintainers = with maintainers; [ lovek323 ];
-      platforms   = platforms.unix;
```

Comment:
I think if we remove `platforms` it will be no longer build on hydra.

Reply:
Agreed. We do the same for buildGoPackage.
It could default to perl.meta.platforms.